examples/ray_serve/train_model.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+from pathlib import Path
+
+import mlflow
+from mlflow.models import Model
+
+from sklearn.datasets import load_iris
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.metrics import mean_squared_error
+from sklearn.utils import shuffle
+
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+
+
+def print_model_info(m_uri, x):
```

Comment:
I know I wrote this code bit. Perhaps call this function `eval_and_print_model_info(...)`